src/angular-fluidbox.js

```diff
@@ -15,7 +15,9 @@ angular.module('fluidbox', [])
         viewportFill: attrs.viewportFill || 0.95
       };
 
-      element.fluidbox(options);
+      $timeout(function () {
+        element.fluidbox(options);
+      });
```

Comment:
Will rather add a watcher on demand than call an apply. I'll work this afternoon to a different solution